server/src/providers/blockchain.ts

```diff
@@ -123,29 +124,34 @@ export class blockchain {
 
       if (this.isClientWallet(fromAddr)) {
         let timestamp = moment.unix(txTime);
-        await transactionsProvider.set({
-          hash: event.blockHash,
-          value: web3Utils.hexToNumber(event.returnValues.value),
-          blockNumber,
-          time: txTime,
-          date: timestamp.format("YYYY-MM-DD"),
-          from: fromAddr,
-          to: toAddr
-        });
-      }
-
-      if (wallets.hasOwnProperty(fromAddr)) {
-        wallets[fromAddr].to = wallets[fromAddr].to + 1;
-        wallets[fromAddr].countTx = wallets[fromAddr].countTx + 1;
-      } else {
-        wallets[fromAddr] = {
+        let date = timestamp.format("YYYY-MM-DD")
+        console.log('set - at ' + date);
+        const amountTX = web3Utils.hexToNumber(event.returnValues.value)
+
+        if (aboutTXs.hasOwnProperty(date)) {
+          aboutTXs[date].amount_txs = aboutTXs[date].amount_txs + amountTX;
+          aboutTXs[date].count_txs = aboutTXs[date].count_txs + 1;
+        } else {
+          aboutTXs[date] = {
+            date,
+            amount_txs: amountTX,
+            count_txs: 1
+          }
+        }
+        if (wallets.hasOwnProperty(fromAddr)) {
+          wallets[fromAddr].to = wallets[fromAddr].to + 1;
```

Comment:
should be from.
refactor from, to to: outTXs, inTXs

Reply:
renamed